docs/common-bugs.rst

```diff
@@ -152,6 +152,16 @@ System upgrades
 :Solution: Anaconda is not doing system upgrades. That is done by dnf-system-upgrade.
     Reassigning to dnf.
 
+Unknown Live installation error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:Issue: The installation of Live stopped with traceback about
+    ``[Errno 2] No such file or directory: systemd-machine-id-setup``.
+:Solution: None known yet. This is probably the first program to run from the installed system.
```

Comment:
I would mention that it could be caused by a corrupted Live ISO or a failing hardware, but we are investigating also other possibilities. We could recommend to check the hardware, download the Live ISO again and rerun the installation.

Reply:
Good idea - I changed it. Thanks!